services/serializers.py

```diff
@@ -10,6 +13,7 @@ class ServiceSerializer(serializers.ModelSerializer):
  
         JSONRenderer().render(UserSerializer(user_instance).data)
     """
+    categoty = CategotySerializer(read_only=True)
```

Comment:
I would remove read_only=True (which here is set to false by default) so this field is included in the PUT/PATCH/POST requests for uploading/updating new services.